src/webgpu/web_platform/reftests/ref/canvas_use_as_image_source_utils.html.ts

```diff
@@ -0,0 +1,138 @@
+import { assert } from '../../../../common/util/util.js';
```

Comment:
filename nit: doesn't need to be `.html.ts` unless it corresponds with a particular `.html` file with that name